### ISSUE TITLE
SubList <neverEmpty> field is invalid or not supported.

### ISSUE THREAD

**Issue by grugknuckle** [oracle/netsuite-suitecloud-sdk#132]
OS: Windows 10
Node.js version: e.g. 12.16.1 LTS
suitecloud-cli version: 1.0.4

### Describe the bug
The CLI validation doesn't handle the <neverEmpty> field of sublists during project validation. 

### To Reproduce
1. use 'suitecloud account:setup` to login into any account. 

2. then use `suitecloud object:import -i` to import into the SDF projcect any custom entry form which has an entry in the sublists tab. For example, in my account, I have a vendor entry form like this

![vendor form screenshot](https://i.ibb.co/wrPZ9Mc/sublists.jpg)

3. Open the .xml file in your editor and search for the subList keyword. Verify that at least ONE of your subList items has the 'neverEmpty' property. For example, with the vendor form shown above

```xml
  <subItems>
    <subList>
      <id>ENTITYMESSAGES</id>
      <label>Messages</label>
      <visible>T</visible>
      <neverEmpty>false</neverEmpty>
    </subList>
  </subItems>
```
4. in the SAME account, execute `suitecloud project:validate`

### Actual Behavior
The CLI raises this error

```bash
An error occurred during custom object validation. (custform_XXXXXX)
Details: The neverEmpty field is invalid or not supported.
```

### Expected Behavior
Objects imported from a working NetSuite instance, should either
1. NOT import xml with unsupported or invalid values. Or, 
2. all values imported from a working account should be valid or supported.

**Comment by colivares82** [comment]
Thanks for reporting this detailed case @grugknuckle , we will investigate and reproduce and I´ll get back to you.

**Comment by stephen-suiterep** [comment]
This behavior occurs in the Eclipse plugin as well. Would love to see this issue resolved!

**Comment by alii-gator** [comment]
@grugknuckle @stephen-suiterep just wanted to give you an update in the reported issue.
This has already bin fixed and we are testing the solution. Hopefully we will role out the fix with upcoming major release.

**Comment by grugknuckle** [comment]
@alii-gator - Thanks for fixing it. I came back to this issue because I happened to read this article in the help.

/app/help/helpcenter.nl?fid=section_N2858591.html

Where it states:

"The Never Empty box is not available for all sublists. Only sublists that are listed in the SuiteScript Records Browser are supported."

Which got me thinking that maybe it could be that the object XML generator is outputing the `<neverEmpty>` tag for unsupported sublists. Or that the code which parses the XML during import isn't taking this exception into account.
